### ISSUE TITLE
Not waiting for page transitions on UWP app

### ISSUE THREAD

**Issue by RafalSkorka** [Microsoft/WinAppDriver#525]
I started to get random failures which are caused by XAML page transitions. Didn't have them before October update but could be a coincidence. 
The issue is WinAppDriver tries to click on the control which is not yet positioned completely due to page transitions. Sometimes it succeeds, but not always.
When it succeeds, it clicks the button while it is still moving up the screen. When it fails, it just misses the button completely.
Is there a way to tell WinAppDriver that it needs to wait for page load?

**Comment by hassanuz** [comment]
Hi @RafalSkorka,

You can try adding Thread.Sleep timers for a quick workaround - at the moment there is nothing from the WinAppDriver server side to help with this.

**Comment by RafalSkorka** [comment]
Yes, that's what I did for now, but I was hoping for some more robust solution.
